static_src/components/user_list.jsx

```diff
@@ -22,19 +27,59 @@ export default class UserList extends React.Component {
     this.setState({users: nextProps.initialUsers});
   }
 
+  _handleDelete = (userGuid, ev) => {
+    this.props.onRemove(userGuid, ev);
+  }
+
   get columns() {
-    return [
+    var columns = [
       { label: 'Name', key: 'username' },
       { label: 'Date Created', key: 'created_at' }
     ];
+
+    if (this.props.onRemove) {
```

Comment:
when will this not be set?. there are similar conditionals elsewhere. if you're planning for something in the future where it won't be set. put a TODO.